packages/fxa-settings/src/pages/Signin/SigninRecoveryCode/en.ftl

```diff
@@ -3,15 +3,10 @@
 ## (provided to the user when they first set up two-step authentication)
 ## when they are unable to sign in with two-step authentication (e.g., Authy, Duo, etc.)
 
-# String within the <span> element appears on a separate line
-# If more appropriate in a locale, the string within the <span>, "to continue to account settings" can stand alone as "Continue to account settings"
-signin-recovery-code-heading-w-default-service = Enter backup authentication code <span>to continue to account settings</span>
-# String within the <span> element appears on a separate line
-# If more appropriate in a locale, the string within the <span>, "to continue to { $serviceName }" can stand alone as "Continue to { $serviceName }"
-# { $serviceName } represents a product name (e.g., Mozilla VPN) that will be passed in as a variable
-signin-recovery-code-heading-w-custom-service = Enter backup authentication code <span>to continue to { $serviceName }</span>
-signin-recovery-code-instruction = Please enter a backup authentication code that was provided to you during two step authentication setup.
-signin-recovery-code-input-label = Enter 10-digit backup authentication code
+signin-recovery-code-heading = Sign in
+signin-recovery-code-sub-heading = Enter backup code
+signin-recovery-code-instruction-v2 = Enter one of the one-time use backup codes you saved during two-step authentication setup.
```

Comment:
Are we no longer using the term "backup authentication code"?

Reply:
Recommendation from UX was to simplify to "backup codes" as we update these flows to add an SMS recovery method. Is there terminology reference material for Mozilla accounts localization?